customer-wallet/src/containers/Vendor/VendorWallet/index.js

```diff
@@ -26,12 +31,42 @@ class VendorWallet extends Component {
     getVendorReviews: PropTypes.func.isRequired
   }
 
-  componentWillMount() {
+  unconfirmed = new WebSocket('wss://socket.blockcypher.com/v1/btc/main')
+  confirmed = new WebSocket('wss://socket.blockcypher.com/v1/btc/main')
+
+  componentDidMount() {
+    const { getVendorReviews } = this.props
+
+    getVendorReviews()
     this.getFxRates()
+
+    this.confirmed.onmessage = (event) => {
+      const data = JSON.parse(get(event, 'data', '{}'))
+      getVendorReviews(address)
```

Comment:
I have the same worry as the getTransactionHistory call on the customer view. This data will be fetched from IPFS in the future and loading it all for all events received will cause the app to slow down a lot. What do you think? @Venumteam correct me if I my understanding is not correct.